mempool/mempool.go

```diff
@@ -1458,6 +1622,14 @@ func (mp *TxPool) pruneStakeTx(requiredStakeDifficulty, height int64) {
 			mp.removeTransaction(tx.Tx, true)
 		}
 	}
+
+	for _, tx := range mp.staged {
+		txType := stake.DetermineTxType(tx.MsgTx())
+		if txType == stake.TxTypeSStx &&
+			tx.MsgTx().TxOut[0].Value < requiredStakeDifficulty {
+			mp.removeTransaction(tx, true)
```

Comment:
Shouldn't you be calling `removeStagedTransaction()` here given you're iterating over `mp.staged`?